src/Microsoft.TestPlatform.Common/ExtensionFramework/TestPluginCache.cs

```diff
@@ -309,35 +309,36 @@ internal IEnumerable<string> DefaultExtensionPaths
     /// <returns>
     /// The <see cref="Dictionary"/>.
     /// </returns>
-    internal Dictionary<string, TPluginInfo> GetTestExtensions<TPluginInfo, TExtension>(string extensionAssembly, bool skipCache = false) where TPluginInfo : TestPluginInformation
+    internal Dictionary<string, TPluginInfo> GetTestExtensions<TPluginInfo, TExtension>(
+        string extensionAssembly,
+        bool skipCache = false) where TPluginInfo : TestPluginInformation
```

Comment:
Nit: Feel free to ignore but as you split parameters to new line, I'd be nice to also split the type constraint.

```suggestion
        bool skipCache = false) 
        where TPluginInfo : TestPluginInformation
```

Reply:
Fixed.